Matey/Controllers/AddNewTransactionViewController.swift

```diff
@@ -0,0 +1,100 @@
+        textFieldFriendUsername.text = ""
+        textFieldLendAmount.text = ""
+        textFieldBorrowAmount.text = ""
+    }
+}
+
+// MARK: - Interface Setup
+
+extension AddNewTransactionViewController: AddNewTransactionViewControllerInterface {
+
+    func setupUI() {
+        title = ConstantsAddNewTransactionVC.title
+        navigationController?.navigationBar.tintColor = ConstantsAddNewTransactionVC.navigationBarTintColor
+
+        textFieldRegistrantsName.placeholder = ConstantsAddNewTransactionVC.textFieldRegistrantsNamePlaceholder
+        textFieldRegistrantsName.textFieldCornerRadius()
+        textFieldRegistrantsName.textFieldLeftPadding()
+
+        textFieldRegistrantsUsername.placeholder = ConstantsAddNewTransactionVC.textFieldRegistrantsUsernamePlaceholder
+        textFieldRegistrantsUsername.textFieldCornerRadius()
+        textFieldRegistrantsUsername.textFieldLeftPadding()
+
+        textFieldFriendUsername.placeholder = ConstantsAddNewTransactionVC.textFieldFriendUsernamePlaceholder
+        textFieldFriendUsername.textFieldCornerRadius()
+        textFieldFriendUsername.textFieldLeftPadding()
+
+        textFieldLendAmount.placeholder = ConstantsAddNewTransactionVC.textFieldLendAmountPlaceholder
+        textFieldLendAmount.keyboardType = ConstantsAddNewTransactionVC.textFieldLendAmountKeyboardType
+        textFieldLendAmount.textFieldCornerRadius()
+        textFieldLendAmount.textFieldLeftPadding()
+
+        textFieldBorrowAmount.placeholder = ConstantsAddNewTransactionVC.textFieldBorrowAmountPlaceholder
+        textFieldBorrowAmount.keyboardType = ConstantsAddNewTransactionVC.textFieldBorrowAmountKeyboardType
+        textFieldBorrowAmount.textFieldCornerRadius()
+        textFieldBorrowAmount.textFieldLeftPadding()
+
+        buttonSaveNewTransaction.backgroundColor = ConstantsAddNewTransactionVC.buttonSaveNewTransactionBackgroundColor
+        buttonSaveNewTransaction.tintColor = ConstantsAddNewTransactionVC.buttonSaveNewTransactionTintColor
+        buttonSaveNewTransaction.layer.cornerRadius = ConstantsAddNewTransactionVC.buttonSaveNewTransactionCornerRadius
+        buttonSaveNewTransaction.setTitle(ConstantsAddNewTransactionVC.buttonSaveNewTransactionTitle, for: .normal)
+    }
+
+    func showAlert(title: String, message: String) {
+        self.showAlertExt(title: title, message: message)
+    }
+
+    func saveTransaction() {
+        guard let textRegistrantsName = textFieldRegistrantsName.text, let textFieldRegistrantsUsername = textFieldRegistrantsUsername.text, let textFieldFriendUsername = textFieldFriendUsername.text, let textFieldLendAmount = Double(textFieldLendAmount.text ?? "0"), let textFieldBorrowAmount = Double(textFieldBorrowAmount.text ?? "0") else { return self.showAlert(title: ConstantsAddNewTransactionVC.messageCouldntSave, message: ConstantsAddNewTransactionVC.messageFillLines)}
+
+        viewModel?.textRegistrantsName = textRegistrantsName
```

Comment:
```suggestion
        viewModel.textRegistrantsName = textRegistrantsName
```
Can you edit the whole file? We now have a value that is not optional.